packages/manager-react-components/src/components/datagrid/datagrid.component.tsx

```diff
@@ -33,6 +34,7 @@ import { FilterWithLabel } from '../filters/interface';
 import { DataGridTextCell } from './text-cell.component';
 import { defaultNumberOfLoadingRows } from './datagrid.contants';
 import './translations';
+import cn from 'classnames';
```

Comment:
```suggestion
import cn from 'classnames';
```

Can you use 'clsx' as Its already installed ?